leptos-fluent-macros/src/lib.rs

```diff
@@ -1467,7 +1467,7 @@ pub fn leptos_fluent(
             }).collect();
 
         let window_navigator_languages_quote = quote! {
-            let languages = window().navigator().languages().to_vec();
+            let languages = ::leptos::leptos_dom::window().navigator().languages().to_vec();
```

Comment:
Ah, you're right! 😅 

Just a minor tweak... could you use `::leptos::window` directly?